sdk/src/test/java/io/opentelemetry/sdk/trace/SpanBuilderSdkTest.java

```diff
@@ -172,6 +172,44 @@ public void setAttribute() {
     }
   }
 
+  @Test
+  public void setAttribute_afterEnd() {
+    Span.Builder spanBuilder = tracerSdk.spanBuilder(SPAN_NAME);
```

Comment:
Do you really need to add 5 attributes for this test? You could have a single item and assert that only that item remains afterwards, and make the test simpler/smaller.

Reply:
Oh, never mind. You are exercising all the attribute types.